### DISCUSSION TITLE
shared tracking domain for mail servers

### DISCUSSION THREAD

**Discussion by ergineroglu** [postalserver/postal#2203] [Help with using Postal]
Hi,

In our use case we will configure multiple mail servers and each server will have a mail sending domain, but we want to give option for each mail sending domain to track open/clicks in a shared domain.

In current test environment we have two domains:

- trendify.co - mail sender domain and mails are coming from offers@trendify.co 
- sgmntify.com - main domain which we will offer our service and want to use for tracking with subdomain track.sgmntify.com 

When we try to send an email from sgmntify.com domain tracking works, but not for trendify.co. How can we force tracking done from a specific domain for a mail server?

I know in ideal situation, tracking should be done for mail sending domain, but we can't ask for our smaller customers to give us a separate custom domain and a ssl certificate for this domain.

Screenshot from the panel: 
![a](https://user-images.githubusercontent.com/5336000/197700382-9dc83990-6550-4069-9282-a7be989b6796.jpg)

**Comment by onurbektas** [comment]
It seems postal wants tracking domain name predefined in postal.yml configuration file. Changing  
_track_domain: track.sgmntfy.com_  to _track_domain: track.trendify.co_ under DNS section  in postal.yml and reloading postal solved the problem for trendify.co domain. But this time, tracking sgmntfy.com failed. I think there should be a way to add multiple tracking domains to postal.

**Reply by willpower232** [reply]
I think you misunderstand how it works, the entries in the config file are your domains to refer to your server and then when you add domains to Postal, you CNAME them to the ones from the config file.

In this case, Postal is sad that `track.trendify.co` is an A record, not a CNAME to the tracking domain Postal thinks it knows.

**Reply by onurbektas** [reply]
Thanks for the answer. Here is the working solution: 
- define  A record for track.sgmntfy.com in DNS server
- set  track_domain: track.sgmntfy.com in postal config file
- define  trck.trendify.co IN CNAME track.sgmntfy.com 
- set  trck.trendify.co as tracing domain in postal web interface.
